livekit-plugins/livekit-plugins-deepgram/livekit/plugins/deepgram/stt.py

```diff
@@ -256,23 +255,27 @@ async def keepalive_task():
 
         async def send_task():
             nonlocal closing_ws
-            # forward inputs to deepgram
-            # if we receive a close message, signal it to deepgram and break.
-            # the recv task will then make sure to process the remaining audio and stop
+
+            # forward audio to deepgram in chunks of 200ms
+            ms200 = self._opts.sample_rate // 5 * self._opts.num_channels
+            audio_bstream = utils.audio.AudioByteStream(
+                sample_rate=self._opts.sample_rate,
+                num_channels=self._opts.num_channels,
+                samples_per_frame=ms200,
+            )
+
             async for data in self._input_ch:
-                if isinstance(data, rtc.AudioFrame):
-                    # TODO(theomonnom): The remix_and_resample method is low quality
-                    # and should be replaced with a continuous resampling
-                    frame = data.remix_and_resample(
```

Comment:
Removed the resampling for now. This shouldn't impact users as the sample_rate property isn't currently exposed